site/docs/extensions/auto-refresh.md

```diff
@@ -76,4 +76,4 @@ toc: true
 
 - **Parameter:** `undefined`
 
-- **Default:** `'Auto Refresh'`
+- **Default:** `function () { return "Auto Refresh" }`
```

Comment:
How will the localize works here?

Reply:
What do you mean ?
I just added missing locales and changed other locales to that format for consistency.